packages/app/components/stackbox/Stackbox.tsx

```diff
@@ -585,9 +589,15 @@ export const Stackbox = () => {
           amount={tokenAmount}
           frequency={frequency}
           startTime={
-            isPastStartDate ? new Date(getDateNowPlus10Mins()) : startDateTime
+            isPastStartDate
+              ? new Date(getDatePlus10Mins(startDateTime))
+              : startDateTime
```

Comment:
This will not work if the user chooses the current date and hour and then only submits the tx 15 minutes after. This will be 5 minutes in the past and tx will fail.